src/modules/Builder/features/ActivitySettings/ScoresAndReports/ScoreContent/ScoreContent.utils.ts

```diff
@@ -34,25 +34,25 @@ const getItemScoreRange = (item: Item) => {
   ) {
     scores = (item as SingleSelectItem | MultiSelectItem).responseValues.options?.reduce(
       (result: number[], option: SingleAndMultipleSelectionOption) => {
-        if (!option.isHidden && option.score) {
-          return [...result, option.score];
+        if (!option.isHidden) {
+          return [...result, option.score as number];
```

Comment:
could you check if `option.score` should have only `number` type? Why do you need to cast?